cmd/restic/cmd_prune.go

```diff
@@ -187,40 +199,54 @@ type packInfoWithID struct {
 	packInfo
 }
 
-// prune selects which files to rewrite and then does that. The map usedBlobs is
-// modified in the process.
-func prune(opts PruneOptions, gopts GlobalOptions, repo restic.Repository, usedBlobs restic.BlobSet) error {
-	ctx := gopts.ctx
+type pruneStats struct {
```

Comment:
I'd like to have a 
```go
MessageType         string  `json:"message_type"` // "summary"
```
field as with the backup command. That should make it easier to extend the output format in the future to e.g. print progress reports

Reply:
Also a very good idea! Is added.